packages/calcite-components/src/components/block/block.tsx

```diff
@@ -378,7 +385,7 @@ export class Block
 
     const headerNode = (
       <div class={CSS.headerContainer}>
-        {this.dragHandle ? <calcite-handle label={heading} /> : null}
+        {this.dragHandle ? <calcite-handle label={heading || label} /> : null}
```

Comment:
Unrelated, but maybe this should use the newer `calcite-drag-handle` at some point? Should we have an issue. @jcfranco

Reply:
Good idea.

Reply:
I think #10382 should cover it 👍